tests/phpunit/StorageExtractorTest.php

```diff
@@ -360,4 +360,38 @@ public function testChangedSince(): void
         );
         self::assertEquals([], $data);
     }
+
+    public function testInfoAction(): void
+    {
+        $temp = new Temp('ex-storage');
+        $temp->initRunFolder();
+        $baseDir = $temp->getTmpFolder();
+        $fs = new Filesystem();
+
+        $configFile = [
+            'action' => 'info',
+            'parameters' => [
+                '#token' => getenv('KBC_TEST_TOKEN'),
+                'url' => getenv('KBC_TEST_URL'),
+            ],
+        ];
+
+        $fs->dumpFile($baseDir . '/config.json', \GuzzleHttp\json_encode($configFile));
+        putenv('KBC_DATADIR=' . $baseDir);
+        $app = new Component(new NullLogger());
+        $result = '';
+        ob_start(function ($content) use (&$result): void {
+            $result .= $content;
+        });
+        $app->run();
+        ob_end_clean();
+
+        $decodeResult = json_decode($result, true);
+
+        self::assertArrayHasKey('projectId', $decodeResult);
```

Comment:
Nebolo by to lepsie porovnavat s konkretnou hodnotou? Tiez by mohla byt cez ENV ... ak by boli niekde v kode omylom vymenene hodnoty, tak tento test to nezisti.

Reply:
jj máš pravdu, to mi nějak uteklo - je to tam